src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -236,7 +241,8 @@ const ChatItemMyTextV2 = ({
     <View style={styles.chatContainer}>
       <View style={handleTextContainerStyle()}>
         {!isContinuous && (
-          <View style={styles.chatTitleContainer}>
+          <View
+            style={[styles.chatTitleContainer, attachments.length > 0 ? {marginBottom: 4} : {}]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a conditional margin to the chatTitleContainer based on whether there are attachments or not is a good way to ensure proper spacing. This change should improve the visual appearance of the chat items when attachments are present.



<!-- This is an auto-generated comment by OSS CodeRabbit -->